src/metabase/models/interface.clj

```diff
@@ -374,6 +374,77 @@
   :args ::define-hydration-method
   :ret  any?)
 
+;;; +----------------------------------------------------------------------------------------------------------------+
+;;; |                                               Toucan 2 Extensions                                              |
+;;; +----------------------------------------------------------------------------------------------------------------+
+
+;; --- transforms methods
+(def tf-metabase-query
```

Comment:
Nitpicky but I think `xform-` is the conventional shorthand for `transform-`. I would have preferred to just write it out the long way tho

Reply:
`s/tf-/transform-/` in 40b6bb2e38